compiler/rustc_middle/src/middle/privacy.rs

```diff
@@ -104,30 +144,78 @@ impl<Id: Hash + Eq + Copy> AccessLevels<Id> {
     }
 
     pub fn set_access_level(&mut self, id: Id, tag: AccessLevel) {
```

Comment:
Looks like this method is never used anymore?

Reply:
I suppose it should be removed after populating effective visibility on rust_privacy cause it's still used there. Also, it's used in librastdoc.